dojo/settings/settings.dist.py

```diff
@@ -1070,6 +1070,7 @@ def saml2_attrib_map_format(dict):
     'Harbor Vulnerability Scan': ['title'],
     'Rusty Hog Scan': ['title', 'description'],
     'StackHawk HawkScan': ['vuln_id_from_tool', 'component_name', 'component_version'],
+    'Hydra Scan': ['title', 'description'],
```

Comment:
Not at this line but in the same file: Deduplication works better, if you set `DEDUPLICATION_ALGORITHM_PER_PARSER` to `DEDUPE_ALGO_HASH_CODE` for this parser.